frontend/nightwatch.conf.js

```diff
@@ -39,7 +34,10 @@ module.exports = {
                 'browserName': 'chrome',
                 'javascriptEnabled': true, // turn off to test progressive enhancement
                 'chromeOptions': {
-                    'args': os.platform() === 'linux' ? ['headless', 'no-sandbox', browserSize] : [browserSize],
+                    "args" : [
+                        "headless",'no-sandbox', 'disable-setuid-sandbox','disable-gpu','disable-dev-shm-usage',
```

Comment:
Some consistency around quotation marks would be good here. Also, I assume there was a reason this was a conditional before. Are we ok to remove this? Have you checked with @kyle-ssg ?

Reply:
yep we don't need selenium any more